website/src/main/resources/microsite/plugins/RuleRefTag.rb

```diff
@@ -0,0 +1,23 @@
+module Jekyll
+  class RuleRefTag < Liquid::Tag
+
+    def initialize(tag_name, docTitle, tokens)
+      super
+      @docTitle, @hash = docTitle.split(',').map(&:strip)
```

Comment:
`@hash` is not used in the implementation, so this can be

```ruby
@docTitle = docTitle.strip
```

Reply:
Done.